packages/web/src/pages/rewards-page/components/modals/ChallengeRewardsModal/AudioMatchingRewardsModalContent.tsx

```diff
@@ -139,68 +118,33 @@ export const AudioMatchingRewardsModalContent = ({
     onNavigateAway()
   }, [challengeName, onNavigateAway, navigateToPage])
 
-  const formatLabel = useCallback((item: any) => {
-    const { label, claimableDate, isClose } = item
-    const formattedLabel = isClose ? (
-      label
-    ) : (
-      <Text>
-        {label}&nbsp;
-        <Text color='subdued'>{claimableDate.format('(M/D)')}</Text>
-      </Text>
-    )
-    return {
-      ...item,
-      label: formattedLabel
-    }
-  }, [])
-
-  const renderCooldownSummaryTable = () => {
-    if (isCooldownChallengesEmpty) return null
-    return (
-      <SummaryTable
-        title={messages.upcomingRewards}
-        items={formatCooldownChallenges(cooldownChallenges).map(formatLabel)}
-        summaryItem={summary}
-        secondaryTitle={messages.audio}
-        summaryLabelColor='accent'
-        summaryValueColor='default'
-      />
-    )
-  }
-
   return (
-    <Flex column gap='2xl'>
-      {isMobile ? (
-        <>
-          {progressDescription}
-          <Paper column shadow='flat' w='100%' css={{ overflow: 'hidden' }}>
-            <Flex justifyContent='center'>{progressReward}</Flex>
-            {progressStatusLabel}
-          </Paper>
-          {renderCooldownSummaryTable()}
-        </>
-      ) : (
-        <>
-          <Paper shadow='flat' direction='column'>
-            <Flex justifyContent='center'>
-              {progressDescription}
-              {progressReward}
-            </Flex>
-            {progressStatusLabel}
-          </Paper>
-          {renderCooldownSummaryTable()}
-        </>
-      )}
-      {challenge?.claimableAmount && challenge.claimableAmount > 0 ? null : (
-        <Button
-          variant='secondary'
-          fullWidth
-          {...ctaButtonProps[challengeName]}
-          onClick={handleClickCTA}
-        />
-      )}
-      {errorContent}
-    </Flex>
+    <ChallengeRewardsLayout
+      description={progressDescription}
+      reward={progressReward}
+      progress={progressStatusLabel}
+      additionalContent={
+        challenge?.cooldown_days && challenge.cooldown_days > 0 ? (
```

Comment:
`!!challenge?.cooldown_days ? ... : ...`

Reply:
do we want to show summary table if empty?

Reply:
oh yup def, sorry for some reason i thought cooldown_days was an array